protocol/contracts/farm/AppStorage.sol

```diff
@@ -74,6 +74,7 @@ contract Account {
         mapping(address => mapping(uint32 => Deposit)) deposits; // A Farmer's Silo Deposits stored as a map from Token address to Season of Deposit to Deposit.
         mapping(address => mapping(uint32 => uint256)) withdrawals; // A Farmer's Withdrawals from the Silo stored as a map from Token address to Season the Withdrawal becomes Claimable to Withdrawn amount of Tokens.
         SeasonOfPlenty sop; // A Farmer's Season Of Plenty storage.
+        mapping(address => mapping(address => uint256)) depositAllowances; // Spender => Silo Token
```

Comment:
I missed an asymmetry here in my initial review:

Deposits is a mapping of: `Token Address => Season of Deposit => Amount`
Allowances is a mapping of: `Spender => Token Address`

In general I'd advocate for as much consistency in structure here as we can muster unless there are technical reasons to structure things differently.

However, given that this BIP is ~2 days from closing and this will require inverting the parameter order for all usages of depositAllowances, I think we let it go and make a note in future documentation.